zeroize/derive/src/lib.rs

```diff
@@ -31,6 +31,18 @@ decl_derive!(
     derive_zeroize
 );
 
+decl_derive!(
+    [ZeroizeOnDrop, attributes(zeroize)] =>
+
+    /// Derive the `ZeroizeOnDrop` trait.
+    ///
+    /// Supports the following attributes:
+    ///
+    /// On the field level:
+    /// - `#[zeroize(skip)]`: skips this field or variant when calling `zeroize()`
+    derive_zeroize_only_drop
```

Comment:
Was this supposed to be `derive_zeroize_on_drop`? I'm not sure what `only` is supposed to mean in this context